config/cloudinary.js

```diff
@@ -0,0 +1,8 @@
+require("dotenv").config();
```

Comment:
`require("dotenv").config();` should only be done once, at the top on `index.js`